front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.ts

```diff
@@ -235,22 +236,40 @@ export class CreateF3XStep1Component implements OnInit, OnDestroy {
       summary.form_type = F3xFormTypes.F3XT;
     }
 
-    this.f3xSummaryService.create(summary, this.formProperties).subscribe((report: F3xSummary) => {
-      if (jump === 'continue') {
-        // Save report to Cash On Hand in the store if necessary by pulling the reports table data.
-        this.reportService.getTableData().subscribe();
+    //Observables are *defined* here ahead of their execution
+    const create = this.f3xSummaryService.create(summary, this.formProperties);
+    // Save report to Cash On Hand in the store if necessary by pulling the reports table data.
+    const tableData = this.reportService.getTableData();
+    const cashOnHand = this.store.select(selectCashOnHand);
```

Comment:
A small nit-pick: the convention is to add a "$" suffix to variables that contain observables. e.g. tableData$, cashOnHand$

Reply:
:+1: